rust/src/router/consumer.rs

```diff
@@ -239,7 +239,7 @@ pub struct ConsumerStat {
     pub packet_count: usize,
     pub byte_count: usize,
     pub bitrate: u32,
-    pub round_trip_time: Option<u32>,
+    pub round_trip_time: Option<f64>,
```

Comment:
This should be `f32` here and below, it is just `float` on C++ side

Reply:
Changed ```f32```. Thanks